apps/back/migrations/Version20230405075639.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230405075639 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
```

Comment:
You should add a description here to have a better history in the future. Old project can have a hundred migrations, it is interesting to know why we added a migration. This is the aim of this function.

Reply:
Thanks, as a comment on line 19 ?

Reply:
Ex: 
```suggestion
        return 'Update user entity to add new field gender';
```

Reply:
Ok, thanks !